package/yast2-storage-ng.changes

```diff
@@ -1,3 +1,11 @@
+-------------------------------------------------------------------
+Fri Nov 16 09:32:05 UTC 2018 - jlopez@suse.com
+
+- Initial proposal tries all possible attempts over each candidate
+  device before switching to another device.
+- Related to bug#1115819 and fate#326573.
```

Comment:
This is indeed related to those bugzilla/fate entries. But also (even more closely) to [bug#1102026](https://bugzilla.suse.com/show_bug.cgi?id=1102026) (which complains about the default proposal not sticking to the first disk) and [bug#1090383](https://bugzilla.suse.com/show_bug.cgi?id=1090383) (which also states the proposal should only try the first disk unless told to try something else).